src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/diff/PrisonerDifferenceService.kt

```diff
@@ -63,6 +64,7 @@ class PrisonerDifferenceService(
     if (prisonerHasChanged(offenderBooking.offenderNo, storedPrisoner)) {
       generateDiffEvent(existingPrisoner, offenderBooking, storedPrisoner)
       generateDiffTelemetry(existingPrisoner, offenderBooking, storedPrisoner)
+      prisonerMovementsEventService.generateAnyMovementEvents(existingPrisoner, storedPrisoner)
     }
```

Comment:
the meat; as well as generating the "update" event send movement events if required

Reply:
I guess this work will take a week or two - slightly worried that somebody will see the new events in Offender Events UI and start using them before they're ready. There are so many consumers from random managed services they're hard to keep track.

Reply:
I'm not necessarily arguing for a feature switch btw - maybe just some BOLD text saying these events are WIP.